build/noderesourcetopology-plugin/Dockerfile.tools

```diff
@@ -26,9 +26,9 @@ RUN mkdir -p $HOME && \
     tar -C /usr/local -xzf go.tar.gz && \
     rm go.tar.gz && \
     # get required golang tools and OC client
-    go get github.com/onsi/ginkgo/ginkgo && \
-    go get golang.org/x/lint/golint && \
-    go get github.com/mattn/goveralls && \
+    go install github.com/onsi/ginkgo/ginkgo && \
```

Comment:
not sure it is needed, but let's leave this alone for now

Reply:
You mean leave it alone the way I have it after the change or not change it and leave it alone as it was originally?